lib/ansible_variables/utils/vars.py

```diff
@@ -82,7 +82,8 @@ def file_occurrences(self, loader: DataLoader):
 
         for ffile in self.files:
             display.vvv("Checking file %s for occurrence of variable %s" % (ffile, self.name))
-            if loader.load_from_file(ffile) and self.name in loader.load_from_file(ffile):
+            fvars = loader.load_from_file(ffile, unsafe=True)
```

Comment:
can we change the variable name from `fvars` to `content`, this would make it more clear.
Good would be also to include a short comment, that `unsafe=True` will be prevent the deepcopy.

Reply:
Sure! No problem